src/lib.rs

```diff
@@ -92,14 +97,13 @@ macro_rules! metadata {
 /// `main()` function of the program.
 ///
 /// ```
-/// use human_panic::setup_panic;
+/// use human_panic::{metadata, setup_panic};
 ///
-/// setup_panic!(Metadata {
-///     name: env!("CARGO_PKG_NAME").into(),
-///     version: env!("CARGO_PKG_VERSION").into(),
-///     authors: "My Company Support <support@mycompany.com>".into(),
-///     homepage: "support.mycompany.com".into(),
-/// });
+/// let mut metadata = metadata!();
```

Comment:
The Builder way has a trouble that we need to catch up `env!("CARGO_PKG_NAME")` in the caller crate, not in this crate.

But we build a macro to make the default and bind to these fields later.

Reply:
Update.